### PR TITLE
VIH-7645

### PR DESCRIPTION
**Pull request by MichaelKeaneathmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs [hunk 1]
```diff
@@ -165,13 +165,23 @@ public void ScrollToTheTopOfThePage()
...
             {
-                JavascriptClick(element);
-                if (Driver.FindElement(oldElement) == null) return;
-                Thread.Sleep(TimeSpan.FromSeconds(1));
+                try 
+                {
+                   Console.WriteLine($"loop {i}");
+                    JavascriptClick(element);
+                    Thread.Sleep(TimeSpan.FromSeconds(1)); 
+                    Driver.WaitUntilVisible(newElement, 1);
```
Comment (ibthmcts) [inline AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:176]: I don't think both wait times are required. Wouldn't **Driver.WaitUntilVisible(newElement, 1);** do the trick?

File: AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs [hunk 2]
```diff
@@ -171,17 +171,12 @@ public void ClickToProgress(By element, By newElement, int timeout = 20)
...
                     Thread.Sleep(TimeSpan.FromSeconds(1)); 
                     Driver.WaitUntilVisible(newElement, 1);
-                    Console.WriteLine($"new page found in loop {i}");
                     return; 
                 }
-                catch (Exception e)
-                {
-                    Console.WriteLine($"new page NOT found in loop {i}");
-                }
+                catch (Exception e) {}
```
Comment (ibthmcts) [inline AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:179]: It's not normally good practice to have empty exceptions. In this case any exception caught is ignored. I think either remove the try catch and have an action in the catch

Comment (MichaelKeaneathmcts) [reply AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:179]: That's intentional, that exception needs be caught and ignored so to continue looping.  I'll change it to two catches, one for webdrivertimeoutexception and another for no suchelementexception.

File: AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs (hunk 1 above)
Comment (MichaelKeaneathmcts) [reply AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:176]: Fixed, I had a reason for doing it this way but maybe a wait will work.

**Review by ibthmcts — APPROVED** [review]

**Pull request merged**